streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1011,7 +1011,7 @@ private void addTaskToStateUpdater(final Task task) {
         } catch (final LockException lockException) {
             // The state directory may still be locked by another thread, when the rebalance just happened.
             // Retry in the next iteration.
-            log.info("Encountered lock exception. Reattempting locking the state in the next iteration.", lockException);
+            log.debug("Encountered lock exception. Reattempting locking the state in the next iteration.");
```

Comment:
If you remove the exception from the log message, we lose the info about to which task this lock exception belongs. So either you add only the message of the `LockException` to the log message or you add the task ID from `task`. I would prefer the former. Something like:
 ```suggestion
             log.debug("Encountered lock exception. Reattempting locking the state in the next iteration. Error message was: {}" lockException.getMessage());
```

Reply:
I agree.

In this case, I believe the exception message is more useful because it also includes whether it is a standby or active task.

Reply:
Good point!

Reply:
Thanks! I've updated.